onnxruntime/core/providers/nnapi/nnapi_builtin/builders/op_builder.cc

```diff
@@ -535,7 +536,12 @@ class BaseOpBuilder : public IOpBuilder {
 };
 
 Status BaseOpBuilder::AddToModelBuilder(ModelBuilder& model_builder, const Node& node) const {
-  ORT_RETURN_IF_NOT(model_builder.IsNodeSupported(node), "Unsupported operator ", node.OpType());
+  OpSupportCheckParams params{
+      model_builder.GetAndroidSdkVer(),
+      model_builder.UseNCHW(),
+  };
+
```

Comment:
Do we need to construct this on every call to AddToModelBuilder?

Reply:
Can create one and store it in ModelBuilder, will address this in next PR